src/test/store/receive-profile.test.js

```diff
@@ -499,59 +496,63 @@ describe('actions/receive-profile', function() {
 
         const { getState } = setup(profile);
         expect(getHumanReadableTracks(getState())).toEqual([
-          'show [thread GeckoMain tab] SELECTED',
+          'show [thread GeckoMain tab]',
           '  - show [thread Other2]',
         ]);
       });
+    });
+  });
 
-      it('should select the first visible thread during changeShowTabOnly action', function() {
```

Comment:
removed one test here since it doesn't apply in our case anymore. (we are not finding the visible thread)